python/cudf_polars/cudf_polars/dsl/utils/aggregations.py

```diff
@@ -151,6 +151,14 @@ def decompose_single_agg(
                 # agg.
                 replace_nulls(col, 0, is_top=is_top),
             )
+        elif agg.name == "mean":
```

Comment:
Would we also consider `MEAN` a candidate for https://github.com/rapidsai/cudf/issues/15852? Or is the post cast fine?

Reply:
Yes `MEAN` should be considered a part of #15852. Although the post cast is probably fine for now.